ceph.spec.in

```diff
@@ -330,6 +330,8 @@ BuildRequires:  rdma-core-devel
 BuildRequires:	liblz4-devel >= 1.7
 # for prometheus-alerts
 BuildRequires:  golang-github-prometheus-prometheus
+# for grafana dashboards
+BuildRequires:  jsonnet
```

Comment:
do we use `jsonnet` if we don't run `make check`, in other words, if user only wants to build ceph is `jsonnet` required? if not, i'd guard by `%if 0%{with make_check}`.

Reply:
Done